README.md

```diff
@@ -17,7 +17,7 @@ This repository works in conjunction with the [GitHub public product roadmap](ht
 
 We encourage you to [open a discussion](https://github.com/github/feedback/discussions) if you have suggestions for how we can improve our products. You don't need to have a solution to the problem you are facing to kick off a discussion. We are hoping to foster productive and collaborative conversations, so please check out [how to give good feedback](https://github.com/github/feedback/discussions/1) if you want some guidance on how to kick off a successful discussion.
 
-Prior to making a new discussion, please take a look at previous discussions to see if someone else has already brought to our attention your suggestions. If you find a similar discussion, reply with additional details or react to signal your support rather than creating a new discussion.
```

Comment:
Prior to making a new discussion, please take a look at previous discussions to see if someone else has already brought to our attention your suggestions. If you find a similar discussion, reply with additional details or react to signal your support rather than creating a new discussion.
[My account.pdf](https://github.com/github/feedback/files/7002343/My.account.pdf)

Reply:
apyderbig